### PR TITLE
docs(getting started): update expo tab.

### PR DESCRIPTION
**Pull request by asafkorem**
Update Expo tab under the "Getting Started" doc, with their official documentation guide for Detox integration and rewriting it a bit.

### CODE REVIEW

File: docs/introduction/partials/_getting-started-expo.md [hunk 1]
```diff
@@ -2,24 +2,16 @@
 
 :::caution Note
 
-Expo support with Detox is entirely a community driven effort. We have no specific support in Detox for Expo applications (ejected or otherwise).
-
+Expo integration with Detox is entirely a community-driven effort.
+We have no special support for Expo projects, and we do not maintain any Expo-specific code or documentation.
```
Comment (noomorph) [inline docs/introduction/partials/_getting-started-expo.md:6]:
```suggestion
There is no special support for Expo projects in Detox, and we do not maintain any Expo-specific code or documentation.
```

File: docs/introduction/partials/_getting-started-expo.md [hunk 2]
```diff
@@ -2,24 +2,16 @@
...
 :::
```
Comment (noomorph) [inline docs/introduction/partials/_getting-started-expo.md:7]: Please add a newline before `:::` – this is a lint style.

File: docs/introduction/partials/_getting-started-expo.md [hunk 3]
```diff
@@ -2,24 +2,16 @@
...
-For support on how to use Detox with Expo, you should contact the Expo team or the Expo community. If you
-are seeing an issue, it is most likely not an issue with Detox itself, but with the Expo runtime or with
-an incorrect Detox setup. Genuine issues with Detox can be submitted to [our issue tracker](https://github.com/wix/Detox/issues)
-as usual.
+If you are experiencing a bug, which you believe is a Detox issue, please [open an issue](https://github.com/wix/Detox/issues) on our GitHub repository.
 
-This introduction section is useful for Expo projects, but the next [Project Setup] article, in particular, is **not enough** to complete the setup with Expo.
+This introduction section is useful for Expo projects, but the next [Project Setup] article, in particular, is **not enough** to complete the setup with Expo. Please follow the [Expo guide](https://docs.expo.dev/build-reference/e2e-tests/) instead.
 
 [Project Setup]: ../project-setup.mdx
```
Comment (noomorph) [inline docs/introduction/partials/_getting-started-expo.md:17]:
```suggestion
[Project Setup]: ../project-setup.mdx
[Expo guide]: https://docs.expo.dev/build-reference/e2e-tests
```

File: docs/introduction/partials/_getting-started-expo.md [hunk 4]
```diff
@@ -2,24 +2,16 @@
...
+Expo integration with Detox is entirely a community-driven effort.
+We have no special support for Expo projects, and we do not maintain any Expo-specific code or documentation.
 :::
 
-There are a few resources on the Internet covering how to set up Detox with Expo apps:
-
-- <https://github.com/fschoenfeldt/clean-expo-detox-testing>
-- <https://toshev.medium.com/setting-up-detox-with-a-expo-managed-react-native-project-89342dce4df>
-- <https://blog.expo.dev/testing-expo-apps-with-detox-and-react-native-testing-library-7fbdbb82ac87>
+There is a tutorial on how to set up Detox with Expo projects, which you can find on Expo official documentation site: [Running E2E tests on EAS Build](https://docs.expo.dev/build-reference/e2e-tests/). It is a great starting point for projects that are using Expo.
```
Comment (noomorph) [inline docs/introduction/partials/_getting-started-expo.md:9]: There is a tutorial on [how to set up Detox with Expo projects][Expo guide], which you can find on the Expo official documentation site. It is a great starting point for projects that are using Expo.

File: docs/introduction/partials/_getting-started-expo.md [hunk 5]
```diff
@@ -2,24 +2,16 @@
...
+For support on how to use Detox with Expo, you should contact the Expo team or the Expo community.
 
-For support on how to use Detox with Expo, you should contact the Expo team or the Expo community. If you
-are seeing an issue, it is most likely not an issue with Detox itself, but with the Expo runtime or with
-an incorrect Detox setup. Genuine issues with Detox can be submitted to [our issue tracker](https://github.com/wix/Detox/issues)
-as usual.
+If you are experiencing a bug, which you believe is a Detox issue, please [open an issue](https://github.com/wix/Detox/issues) on our GitHub repository.
 
-This introduction section is useful for Expo projects, but the next [Project Setup] article, in particular, is **not enough** to complete the setup with Expo.
+This introduction section is useful for Expo projects, but the next [Project Setup] article, in particular, is **not enough** to complete the setup with Expo. Please follow the [Expo guide](https://docs.expo.dev/build-reference/e2e-tests/) instead.
```
Comment (noomorph) [inline docs/introduction/partials/_getting-started-expo.md:15]: I suggest removing the last paragraph. Let's add this reminder to the **Project Setup** tutorial instead – it already has an admonition:

image

**Review by noomorph — CHANGES_REQUESTED** [review]

File: docs/introduction/partials/_getting-started-expo.md (hunk 2 above)
Comment (asafkorem) [reply docs/introduction/partials/_getting-started-expo.md:7]: Done :+1:

File: docs/introduction/partials/_getting-started-expo.md (hunk 5 above)
Comment (asafkorem) [reply docs/introduction/partials/_getting-started-expo.md]: done 👍🏼

**Pull request merged**